src/utils.py

```diff
@@ -0,0 +1,37 @@
+from . import constants 
+from viam.proto.common import Vector3
+
+def init_accelerometer(i2cbus):
```

Comment:
more comments on what these functions do - can you use pylint to see what is suggested for formatting and docstrings?